scalyr_agent/configuration.py

```diff
@@ -1781,6 +1785,15 @@ def __verify_main_config_and_apply_defaults(
             separators=[None, ","],
             env_aware=True,
         )
+        self.__verify_or_set_optional_array_of_strings(
+            config,
+            "k8s_include_namespaces",
+            ["*"],
```

Comment:
For consistency with `k8s_ignore_namespaces` maybe we should define `Configuration.DEFAULT_K8S_INCLUDE_NAMESPACES` as a constant value set to `["*"]`

Reply:
Done.